e2e/tenant_resources_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/api/networking/v1"
+	v1 "k8s.io/api/networking/v1"
```

Comment:
Please, use the alias `networkingv1`

Reply:
these imports are duplicated, please jus use `networkingv1` and drop `v1 "k8s.io/api/networking/v1"`

Reply:
sorry, seems like my VSCode is adding this. Fixed